app/scripts/views/sign_up.js

```diff
@@ -192,7 +193,10 @@ function (Cocktail, _, p, BaseView, FormView, Template, Session, AuthErrors,
     },
 
     suggestEmail: function () {
-      mailcheck(this.$el.find('.email'), this.metrics, this.translator, this.window.location.search);
+      console.log(this._able);
+      console.log(this._able.choose('mailcheckEnabled'));
```

Comment:
@dannycoates from IRC:
```
[10:11:58]  <vladikoff>	stomlinson, Danny uses `subject.sessionId` in the functions here: https://github.com/dannycoates/able/blob/master/doc/cookbook.md#ab-test-a-setting do we have something on the content server that we already have for that? Otherwise i am thinking of adding a random sessionId saved in localStorage
[10:13:25]  <stomlinson>	vladikoff: danny and I discussed adding a notion of the sessionId internally to able to handle this. I tried using localStorage.testId at one point. I just generated a UUID, saved it to localStorage and used that.
[10:13:37]  <stomlinson>	vladikoff: perhaps danny has added that by now, not sure.
[10:13:56]  <vladikoff>	stomlinson, ah
```

Is it something that can be added to able or should I work on adding the UUID?